react-native-community/cli
# feat: opt-in pod install with ios commands

Summary:
---------
Adding new config parameter `project.ios.automaticPodsInstallation`.

For existing apps, it means developers can opt-in for automatic CocoaPods installation when running `run-ios` or `build-ios`. Some projects are running on non-standard setups or pods installation is simply not desired.

For the users using `init` command starting from 0.73, it will automatically create the `react-native.config.js` file with this param set to `true`. File is created by CLI after processing the template.
 



Test Plan:
----------
1. Create a project with `npx react-native init` 
2. Link the CLI packages following the Contributing guide
3. `run-ios` and make sure CocoaPods step is skipped
4. Create `react-native.config.js` file and set `project.ios.automaticPodsInstallation` to `true`
5. `run-ios` again and confirm that CocoaPods are getting installed
6. Create another project with the local CLI `node ./path/to/cli init`
7. When prompted about pods installation, press `n`
8. Once project is set, confirm `react-native.config.js` file exists
9. Run `run-ios` and verify pods are getting installed


Checklist
----------
